File: src/main/webapp/js/service/infoService.js
//服务层
app.service('infoService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../info/findAll');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../info/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../info/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../info/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../info/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../info/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../info/search?page='+page+"&rows="+rows, searchEntity);
	}

	this.searchAve = function (page,rows,searchEntity) {
		return $http.post('/info/searchGroupAve?page='+page+"&rows="+rows, searchEntity);
	}
});

File: src/main/java/top/axbt/pta/service/InfoService.java
package top.axbt.pta.service;
import java.util.List;

import top.axbt.pta.entity.PageResult;
import top.axbt.pta.domain.TbInfo;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface InfoService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbInfo> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbInfo info);
	
	
	/**
	 * 修改
	 */
	public void update(TbInfo info);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbInfo findOne(Integer id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Integer[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbInfo info, int pageNum, int pageSize);


	List<TbInfo> findZhuXiao();

	List<TbInfo> findJinChen();
	
}
